addons/web/static/src/webclient/user_menu/user_menu_items.js

```diff
@@ -129,7 +129,7 @@ function logOutItem(env) {
         callback: () => {
             browser.location.href = route;
         },
-        sequence: 70,
```

Comment:
your linter uses a wrong config

Reply:
the freeze script went crazy it seems

Reply:
@aab-odoo it's all your fault. The precommit thing did bad things